File: Ent_UsAdmin/Ent_UsAdmin/TiposDocumentosSolicitud.vb
Imports System.Data.OleDb
Imports System.Data.SqlClient

Public Class TiposDocumentosSolicitud
    Private Sub TiposDocumentosSolicitud_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        cargarDocumentos()
    End Sub


    Public Sub cargarDocumentos()
        dtimpuestos.Rows.Clear()
        Try
            Dim strimpuestos As String
            iniciarconexionempresa()

            strimpuestos = "select * from TiposdeDocumentosSolicitud"

            Dim ejec = New SqlCommand(strimpuestos)
            ejec.Connection = conexionempresa
            Dim id, nombre, valor, factor, tipo As String

            Dim myreaderimpuestos As SqlDataReader = ejec.ExecuteReader()
            While myreaderimpuestos.Read
                id = myreaderimpuestos("id")
                nombre = myreaderimpuestos("nombre")

                dtimpuestos.Rows.Add(id, nombre, myreaderimpuestos("tipo"))
            End While
            myreaderimpuestos.Close()
        Catch ex As Exception
            MessageBox.Show(ex.Message)
        End Try

    End Sub

    Private Sub BunifuThinButton21_Click(sender As Object, e As EventArgs) Handles BunifuThinButton21.Click
        cargarDocumentos()
    End Sub

    Private Sub btn_agregar_Click(sender As Object, e As EventArgs) Handles btn_agregar.Click
        AgregarTipoDocumentoSolicitud.Show()
    End Sub
End Class

File: Ent_UsAdmin/Catalogos/TiposDocumentosSolicitud.cs
using System;
using System.Data.SqlClient;
using System.Windows.Forms;
using Microsoft.VisualBasic.CompilerServices;

namespace ConfiaAdmin
{

    public partial class TiposDocumentosSolicitud
    {
        public TiposDocumentosSolicitud()
        {
            InitializeComponent();
        }
        private void TiposDocumentosSolicitud_Load(object sender, EventArgs e)
        {
            cargarDocumentos();
        }


        public void cargarDocumentos()
        {
            dtimpuestos.Rows.Clear();
            try
            {
                string strimpuestos;
                Module1.iniciarconexionempresa();

                strimpuestos = "select * from TiposdeDocumentosSolicitud";

                var ejec = new SqlCommand(strimpuestos);
                ejec.Connection = Module1.conexionempresa;
                string id, nombre, valor, factor, tipo;

                var myreaderimpuestos = ejec.ExecuteReader();
                while (myreaderimpuestos.Read())
                {
                    id = Conversions.ToString(myreaderimpuestos["id"]);
                    nombre = Conversions.ToString(myreaderimpuestos["nombre"]);

                    dtimpuestos.Rows.Add(id, nombre, myreaderimpuestos["tipo"]);
                }
                myreaderimpuestos.Close();
            }
            catch (Exception ex)
            {
                MessageBox.Show(ex.Message);
            }

        }

        private void BunifuThinButton21_Click(object sender, EventArgs e)
        {
            cargarDocumentos();
        }

        private void btn_agregar_Click(object sender, EventArgs e)
        {
            My.MyProject.Forms.AgregarTipoDocumentoSolicitud.Show();
        }
    }
}